Makefile

```diff
@@ -76,3 +76,10 @@ doxy:
 doxyshow: doxy
 	xdg-open doc/doxy/index.html
 
+formatting: uncrustify
+
+clang-format:
+	find . -name "*.cc" -o -name "*.h" | xargs clang-format -style=file -i
+
+uncrustify: clang-format
```

Comment:
This seems to imply that a run of uncrustify needs a run of clang-format as a pre-requisite. I think both running "only uncrustify" and running "first clang-format and then uncrustify" should be valid alternatives, depending on how tightly you want your code to be brought in line with the code style.